src/containers/ProfileApiKeys/ProfileApiKeys.pcss

```diff
@@ -0,0 +1,25 @@
+td {
```

Comment:
better to add a className to this block and not define styles directly to td element, it can break styles in unexpected places